TypescriptExample/companyReport.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
This looks like the compiled typescript file.  Should we commit this?

Reply:
I will just make another pull request with only typescript files.

Reply:
I had deleted the compiled typescript files.